.travis.yml

```diff
@@ -1,3 +1,51 @@
-language: node_js
-node_js:
-  - "lts/*"
\ No newline at end of file
+matrix:
+    include:
+        - language: node_js
+          node_js:
+            - lts/*
+          script:
+              - yarn build:node
+          cache:
+            yarn: true
+            directories:
+              - 'node_modules'
+
+          deploy:
+            provider: script
+            skip_cleanup: true
+            script: rsync -r --delete-after --quiet $TRAVIS_BUILD_DIR/admin/dist/ btsnake@185.26.156.85:etc/lindy-website/admin/dist/ && rsync -r --delete-after --quiet $TRAVIS_BUILD_DIR/static/ btsnake@185.26.156.85:etc/lindy-website/static/
+
+            on:
+              branch: deploy
+
+        - language: rust
+          rust:
+              - nightly
+          install: rustup target add x86_64-unknown-linux-musl --toolchain=nightly
+          script:
+              - cargo build --release --target x86_64-unknown-linux-musl
+          cache: cargo
+
+          deploy:
+            provider: script
+            skip_cleanup: true
+            script: ssh btsnake@185.26.156.85 supervisorctl stop lindy-website && rsync -r --delete-after --quiet $TRAVIS_BUILD_DIR/target/x86_64-unknown-linux-musl/release/lindyhop-aachen btsnake@185.26.156.85:bin/lindy-website && ssh btsnake@185.26.156.85 supervisorctl start lindy-website
+            on:
+              branch: deploy
+
+install: yarn --frozen-lockfile
+
+addons:
+  ssh_known_hosts:
+  - 185.26.156.85
+  apt:
+    packages:
+      # For building MUSL static builds on Linux.
+      - musl-tools
+
+before_install:
+- openssl aes-256-cbc -K $encrypted_c628346eb8fe_key -iv $encrypted_c628346eb8fe_iv
```

Comment:
Eine Alternative wäre es, die keys direkt in Travis als env vars zu speichern. Dann würden sie nicht (wenn auch verschlüsselt) im repo liegen.